packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -572,9 +572,7 @@ export class DropdownBase extends BaseComponent<IDropdownInternalProps, IDropdow
       this._async.requestAnimationFrame(() => {
         const selectedIndices = this.state.selectedIndices;
         if (selectedIndices && selectedIndices[0] && !this.props.options[selectedIndices[0]].disabled) {
-          const element: HTMLElement = getDocument()!.querySelector(
-            `#${this._id}-list${selectedIndices[0]}`
-          ) as HTMLElement;
+          const element: HTMLElement = getDocument()!.querySelector(`#${this._id}-list${selectedIndices[0]}`) as HTMLElement;
```

Comment:
Have no idea why my code editor along with prettier is doing this change. ❓

Reply:
Must have been some change in the Prettier config since this was last saved, although I thought the change was to shorter line lengths. It's a mystery.